nifi-nar-bundles/nifi-hbase-bundle/nifi-hbase-processors/src/main/java/org/apache/nifi/hbase/PutHBaseRecord.java

```diff
@@ -49,10 +51,12 @@
 @InputRequirement(InputRequirement.Requirement.INPUT_REQUIRED)
 @Tags({"hadoop", "hbase", "put", "record"})
 @CapabilityDescription("Adds rows to HBase based on the contents of a flowfile using a configured record reader.")
+@ReadsAttribute(attribute = "restart.index", description = "Reads restart.index when it needs to replay part of a record set that did not get into HBase.")
+@WritesAttribute(attribute = "restart.index", description = "Writes restart.index when a batch fails to be insert into HBase")
 public class PutHBaseRecord extends AbstractPutHBase {
 
     protected static final PropertyDescriptor ROW_FIELD_NAME = new PropertyDescriptor.Builder()
-            .name("Row Identifier Field Name")
+            .name("Row Identifier Record Path")
```

Comment:
I think I confused things here with my previous comment... If we made this property a record path then it requires more code later on to evaluate the record path against the record, which we aren't doing right now, so lets put this back to "Row Identifier Field Name".  Sorry for the confusion.